src/main/java/com/google/alpollo/Song.java

```diff
@@ -12,6 +12,10 @@ public Song(String artist, String name, String album) {
     this.album = album;
   }
 
+  public Song(Song song) {
+    this(song.artist, song.name, song.album);
+  }
+
```

Comment:
Why did you create this constructor? Is it ever used?

Reply:
Yes, I have a constructor of SongHead with argument Song because it is easier and I shouldn't to split to fields every time.